### PR TITLE
add feature to finish the app

### PR DESCRIPTION
**Pull request by mario222k**
update build config
remove app example tests
add feature to finish the app
update example app
set version to 1.1.0

### CODE REVIEW

**Review by thiagoferreirap — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -56,5 +56,14 @@ Of course you can register as many listeners as you want.
 
 To stop listening use `LastWords.unregister(listener)` - you don't want to create nasty memory leaks, do you?
 
+If you want to kill your app process with _lastwords_ you can use:
+
+ ```kotlin
+LastWords.finishApp(killDelay)
+ ```
+ This will finish all Activities, then wait for the given delay and kill your process (if no other Activity is started meanwhile).
```
Comment (AndroidGuyDD) [inline README.md]: is started in the meantime.

File: README.md [hunk 2]
```diff
@@ -56,5 +56,14 @@ Of course you can register as many listeners as you want.
...
+ An example would be to notify the user that changes take place after App restart and you show him an option to close the app.
```
Comment (AndroidGuyDD) [inline README.md]: ...that changes took place...

File: library/build.gradle [hunk 3]
```diff
@@ -14,8 +14,8 @@ android {
     defaultConfig {
         minSdkVersion 17
         targetSdkVersion 27
-        versionCode 101
-        versionName "1.0.1"
+        versionCode 100
```
Comment (AndroidGuyDD) [inline library/build.gradle]: not 110..at least not less then last version

**Review by AndroidGuyDD — CHANGES_REQUESTED** [review]
since it's public change log some english changes ^^

File: library/build.gradle (hunk 3 above)
Comment (mario222k) [reply library/build.gradle]: true

File: README.md (hunk 1 above)
Comment (mario222k) [reply README.md]: thx

**Review by thiagoferreirap — APPROVED** [review]

**Review by AndroidGuyDD — APPROVED** [review]

**Pull request merged**